javapeople.yaml

```diff
@@ -356,6 +356,10 @@
   twitter: "@nicolas_frankel"
   fediverse: "@frankel@mastodon.top"
   github: "nfrankel"
+- name: "NYJavaSig"
+  twitter: "@nyjavasig"
+  fediverse: "@nyjavasig@fosstodon.org"
+  github: "-"
```

Comment:
Can you please remove the GitHub property? Otherwise we will render a invalid link. Thanks!